dom.bs

```diff
@@ -5872,6 +5896,10 @@ dictionary ShadowRootInit {
   required ShadowRootMode mode;
   boolean delegatesFocus = false;
 };
+
+dictionary GetInnerHTMLOptions {
+  boolean includeShadowRoots = true;
```

Comment:
At the very least this should say OpenShadowRoots, but it seems better to me if we did not have a separate API here for open and closed.

Reply:
I've added the "rest" of this interface, as explained [here](https://github.com/mfreed7/declarative-shadow-dom/blob/master/README.md#serialization) in the explainer. In particular, the section just below that, [Closed shadow roots](https://github.com/mfreed7/declarative-shadow-dom/blob/master/README.md#closed-shadow-roots) goes into the details of the interface. You essentially get two knobs. `includeShadowRoots` is an opt-in for *any* shadow roots to be serialized. The second input, `closedRoots` is a list of closed shadow roots that you would like to be included. If a closed shadow root is encountered that is not in the list, it will be skipped to preserve encapsulation.

I am going to add the corresponding change to my HTML spec PR.

LMK if the above clarifies the situation, or if you have further questions.